status/src/main/java/com/networknt/status/Status.java

```diff
@@ -258,7 +258,7 @@ public String toString() {
             if (showDescription) {
                 sb.append("\",\"description\":\"" + getDescription());
             }
-            if (showMetadata) {
+            if (showMetadata && getMetadata() != null) {
                 try {
                     sb.append("\",\"metadata\":\"" + Config.getInstance().getMapper().writeValueAsString(getMetadata()));
```

Comment:
When we convert the metadata map to the string, is the final result still can be parsed as a JSON?

Reply:
now the metadata is appending a JSON instead of a string, and it can be parsed back.